oss-cloud-backend/services/utility.js

```diff
@@ -0,0 +1,29 @@
+
+// function to check if arrays are equal
+function arraysEqual(_arr1, _arr2) {
+  if (!Array.isArray(_arr1) || !Array.isArray(_arr2)
+   || _arr1.length !== _arr2.length) { return false; }
+
+  const arr1 = _arr1.concat().sort();

```

Comment:
This will mutate the original array.
We can introduce the utility library. Lodash's [isEqual](https://lodash.com/docs/4.17.15#isEqual) can be used here.

Reply:
That's a great idea.